<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Carbon Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sellers file</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>671</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buyers file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sellerBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>591</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="buyerBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>591</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCarbonCalculator">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChoose_sellers"/>
    <addaction name="actionChoose_buyers"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Info"/>
   </widget>
   <widget class="QMenu" name="menuCompute">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionCompute_Sellers"/>
    <addaction name="actionCompute_Buyers"/>
   </widget>
   <addaction name="menuCarbonCalculator"/>
   <addaction name="menuCompute"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose_sellers">
   <property name="text">
    <string>Choose sellers</string>
   </property>
  </action>
  <action name="actionChoose_buyers">
   <property name="text">
    <string>Choose buyers </string>
   </property>
  </action>
  <action name="actionShow_Info">
   <property name="text">
    <string>Show Info</string>
   </property>
  </action>
  <action name="actionCompute_Sellers">
   <property name="text">
    <string>Compute Sellers</string>
   </property>
  </action>
  <action name="actionCompute_Buyers">
   <property name="text">
    <string>Compute Buyers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
